vajram/vajram-samples/src/main/java/com/flipkart/krystal/vajram/samples/benchmarks/calculator/Formula.java

```diff
@@ -17,27 +17,27 @@ public abstract class Formula extends ComputeVajram<Integer> {
   public static final String ID = "formula";
 
   @Resolve(value = "sum", inputs = "number_one")
-  public static int adderNumberOne(@BindFrom("p") int p) {
+  public int adderNumberOne(@BindFrom("p") int p) {
```

Comment:
All resolver and VajramLogic methods must be static to avoid misuse of class level fields

Reply:
Done